client/lib/signup/step-actions/index.js

```diff
@@ -166,6 +166,16 @@ function getNewSiteParams( {
 	// when segment and vertical values are not sent. Check pbAok1-p2#comment-834.
 	const shouldUseDefaultAnnotationAsFallback = true;
 
+	let shouldEnableFse = selectedDesign?.is_fse;
+	// If the user didn't select any design, then we make sure
+	// the default theme is used to determine the `shouldEnableFse`'s value.
+	if ( ! selectedDesign ) {
+		const designs = getAvailableDesigns();
+		const themeSlug = theme.replace( 'pub/', '' );
```

Comment:
Quick, non-blocking callout. It might be worthwhile to consider replacing 'premium/' as well (or leave a comment to do so in the future). There's a new dotcom initiative to create premium block-based themes p58i-aBz-p2. If we ever decide to feature said premium themes, it looks like we'll need to make a corresponding update here.